R/parse_url.R

```diff
@@ -1,18 +1,23 @@
-#' Creates Shinylive url for the app code.
+#' Creates Shinylive url based on app code.
 #'
 #' @importFrom jsonlite unbox toJSON
 #' @importFrom lzstring compressToEncodedURIComponent
 #'
 #' @param code (`character(1)`) A string with app code.
+#' @param mode (`character(1)`) A string with mode. One of "app" or "editor".
```

Comment:
When not defined, the default value for `mode` argument would be `app`. Should this be mentioned here?

Reply:
Added!